### DISCUSSION TITLE
How to Add Time Zone Abbreviation to Name Template String?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [AlexandreRouma/SDRPlusPlus#1357] [Q&A]
I have this recording name template string:
```
$t_$f_$y-$M-$d_$h-$m-$s
```

How do I add the time zone to this so that I can get this e.g.?:
```
audio_99300000Hz_2024-03-13_14-42-14_UTC.wav
```

And yes; technically UTC isn't a **time zone** but a **time standard**.

**Comment by AlexandreRouma** [comment]
the time is not UTC, it's local time

**Reply by AlexLandherr** [reply]
In my case my "local time" is set to UTC. I just need the placeholder for the time zone abbreviation please.

**Reply by AlexandreRouma** [reply]
just add the text in the template...

**Reply by AlexLandherr** [reply]
So there's no "$[tz placeholder]" for SDR++ at all?

If anyone else wants this command and isn't on UTC they shouldn't have to put in their abbreviation manually. This seems like a reasonable ask.

**Reply by AlexandreRouma** [reply]
Do you change your computer's timezone so frequently to the point that typing in the 3 letters of "UTC" is more complicated than typing in the three letters of something like "$tz"? There is no portable way of getting a computer's timezone outside of a "GMT+/-n" format, there are variations even between linux distros on how this information should be queried. I'm not adding 50 to 100 lines of code for something nobody fundamentally needs.

**Reply by AlexLandherr** [reply]
Ok, I understand. Just wanted to know if.
